### ISSUE TITLE
Some error in 'base_attack.py'

### ISSUE THREAD

**Issue by byerose** [DSE-MSU/DeepRobust#76]
Hi. There seems to be some error in file `base_attack.py`.  I didn't find the definition of x/y.

> 64 `        elif type(x).__name__ == 'ndarray':`
> 74             `label = torch.tensor(y)`
> 
Bro, you'd better check it.
